syntax = "proto3";

service RegistrationService {
  rpc RegisterUser (RegisterRequest) returns (RegisterResponse);
}

message RegisterRequest {
  string username = 1;
  string password = 2;
}

message RegisterResponse {
  bool success = 1;
  string message = 2;
}